### PR TITLE
changed fluentd in files to collector

### PR DESCRIPTION
**Pull request by mburke5678**
Removing `fluentd` with `collector` in Logging files to make more generic

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill @richm FYI. I've wanted to make this change for a while. it seems to make sense to remove _fluentd_ from the file names, especially where we want to move away from the EFK concept.

**Comment by mburke5678** [comment]
cc @vikram-redhat

File: modules/cluster-logging-collector-external.adoc [hunk 1]
```diff
@@ -58,7 +58,7 @@ $ oc edit configmap/fluentd -n openshift-logging
 . Add certificates to be used in `secure-forward.conf` to the existing
 secret that is mounted on the Fluentd pods. The `your_ca_cert` and
 `your_private_key` values must match what is specified in `secure-forward.conf`
-in `configmap/logging-fluentd`:
+in `configmap/logging-collector`:
 +
```
Comment (richm) [inline modules/cluster-logging-collector-external.adoc]: in 3.11, this is still called `configmap/logging-fluentd` - in 4.x it is called `configmap/fluentd` - we do not yet have a generic `configmap/collector`

File: modules/cluster-logging-configuring-image-about.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ $ oc -n openshift-logging set env deployment/cluster-logging-operator --list | g
 
 ----
 ELASTICSEARCH_IMAGE=registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2 <1>
-FLUENTD_IMAGE=registry.redhat.io/openshift4/ose-logging-fluentd:v4.2 <2>
+FLUENTD_IMAGE=registry.redhat.io/openshift4/ose-logging-collector:v4.2 <2>
```
Comment (richm) [inline modules/cluster-logging-configuring-image-about.adoc]: this is still `fluentd`

File: modules/cluster-logging-external-syslog.adoc [hunk 3]
```diff
@@ -30,7 +30,7 @@ spec:
     spec:
       containers:
         - name: fluentd
-          image: 'registry.redhat.io/openshift4/ose-logging-fluentd:v4.1'
+          image: 'registry.redhat.io/openshift4/ose-logging-collector:v4.1'
```
Comment (richm) [inline modules/cluster-logging-external-syslog.adoc]: this is still `fluentd`

File: release_notes/ocp-4-1-release-notes.adoc [hunk 4]
```diff
@@ -1013,7 +1013,7 @@ openshift4/ose-libvirt-machine-controllers
 openshift4/ose-logging-curator5
 openshift4/ose-logging-elasticsearch5
 openshift4/ose-logging-eventrouter
-openshift4/ose-logging-fluentd
+openshift4/ose-logging-collector
```
Comment (richm) [inline release_notes/ocp-4-1-release-notes.adoc]: this is still `fluentd`

File: modules/cluster-logging-collector-external.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/cluster-logging-collector-external.adoc]: @richm Fantastic. Thanks. Find/replace overkill. I will fix these. Thank you!!

**Review by richm — APPROVED** [review]
/lgtm

**Comment by mburke5678** [comment]
@anpingli PTAL

**Pull request merged**